### PR TITLE
add 4.13.0.Final and 12.13.0.GA to tools.jboss.org

### PR DESCRIPTION
**Pull request by sbouchet**
Signed-off-by: Stephane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

File: _config/products.yml [hunk 1]
```diff
@@ -140,9 +140,35 @@ devstudio:
...
+    simrel-2019-12:
+      12.14.x.Nightly:
         update_site_url: https://devstudio.redhat.com/12/snapshots/updates/
+    simrel-2019-09:
+      12.13.0.GA:
+        archived: false
+        update_site_url: https://devstudio.redhat.com/12/stable/updates/
+        marketplace_url: https://marketplace.eclipse.org/node/1616973
+        marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1616973
+        release_date: 2019-11-04
```
Comment (sbouchet) [inline _config/products.yml]: @jeffmaury  : waiting for official release date ?

File: _config/products.yml [hunk 2]
```diff
@@ -140,9 +140,35 @@ devstudio:
...
+        supported_devstudio_is_version: 12.12.0.GA
```
Comment (sbouchet) [inline _config/products.yml]: @jeffmaury : not sure about the version here ..

File: _config/products.yml [hunk 3]
```diff
@@ -1454,9 +1480,29 @@ jbt_core:
...
   streams:
-    simrel-2019-09:
-      4.13.x.Nightly:
+    simrel-2019-12:
+      4.14.x.Nightly:
         update_site_url: http://download.jboss.org/jbosstools/photon/snapshots/updates/
+    simrel-2019-09:
+      4.13.0.Final:
+        archived: false
+        release_date: 2019-11-04
```
Comment (sbouchet) [inline _config/products.yml]: @jeffmaury : same as above.

File: _config/products.yml (hunk 2 above)
Comment (jeffmaury) [reply _config/products.yml]: It's fine

File: _config/products.yml [hunk 4]
```diff
@@ -140,9 +140,35 @@ devstudio:
...
+            file_size: 841MB
+          - name: JBoss Central Zip
+            url: http://www.jboss.org/download-manager/content/origin/files/sha256/b2/b251a80eed5bd63f0c6c07ae5bc216f0381f0d8e90718e660fbaf427751265bb/codereadystudio-12.13.0.GA-updatesite-central.zip
+            file_size:  85MB
+          - name: Target Platform for Red Hat CodeReady Studio
+            url: https://devstudio.redhat.com/static/12/stable/updates/core/devstudio-12.13.0.GA-target-platform.zip
+            file_size: 839MB
     simrel-2019-06:
       12.12.0.GA:
         archived: false
```
Comment (jeffmaury) [inline _config/products.yml]: Should be archived now

File: _config/products.yml [hunk 5]
```diff
@@ -1454,9 +1480,29 @@ jbt_core:
...
+      4.14.x.Nightly:
         update_site_url: http://download.jboss.org/jbosstools/photon/snapshots/updates/
+    simrel-2019-09:
+      4.13.0.Final:
+        archived: false
+        release_date: 2019-11-06
+        supported_jbt_is_version: 4.13.0.Final
+        blog_announcement_url: /blog/12.13.0.ga.html
+        documentation_url: http://docs.jboss.org/tools/
+        update_site_url: http://download.jboss.org/jbosstools/photon/development/updates/
```
Comment (jeffmaury) [inline _config/products.yml]: Why development ? Should be stable.

File: _config/products.yml [hunk 6]
```diff
@@ -140,9 +140,35 @@ devstudio:
   name: Red Hat CodeReady Studio
   url_path_fragment: devstudio
   streams:
-    simrel-2019-09:
-      12.x.Nightly:
+    simrel-2019-12:
```
Comment (jeffmaury) [inline _config/products.yml]: You also need to define simrel-2019-12 at the eclipse node level

File: _config/products.yml (hunk 2 above)
Comment (jeffmaury) [reply _config/products.yml]: Should be 12.11.0.GA

File: _config/products.yml [hunk 7]
```diff
@@ -1454,9 +1480,29 @@ jbt_core:
...
-    simrel-2019-09:
-      4.13.x.Nightly:
+    simrel-2019-12:
+      4.14.x.Nightly:
         update_site_url: http://download.jboss.org/jbosstools/photon/snapshots/updates/
+    simrel-2019-09:
+      4.13.0.Final:
+        archived: false
+        release_date: 2019-11-06
+        supported_jbt_is_version: 4.13.0.Final
```
Comment (jeffmaury) [inline _config/products.yml]: Should be 4.11.0.final

File: _config/products.yml [hunk 8]
```diff
@@ -1454,9 +1480,29 @@ jbt_core:
...
+            sha256_url: http://download.jboss.org/jbosstools/static/photon/development/updates/core/jbosstools-4.13.0.Final-updatesite-core.zip.sha256
+          - name: Sources zip of all JBoss Core Tools
+            file_size: 201MB
+            url: http://download.jboss.org/jbosstools/static/photon/development/updates/core/jbosstools-4.13.0.Final-src.zip
+            sha256_url: http://download.jboss.org/jbosstools/static/photon/development/updates/core/jbosstools-4.13.0.Final-src.zip.sha256
+          - name: Release Notes
+            url: https://tools.jboss.org/documentation/whatsnew/index.html
     simrel-2019-06:
       4.12.0.Final:
         archived: false
```
Comment (jeffmaury) [inline _config/products.yml]: Should be archived now

File: _config/products.yml (hunk 6 above)
Comment (sbouchet) [reply _config/products.yml]: done.

**Comment by sbouchet** [comment]
@jeffmaury : fixed PR according to review.

**Pull request merged**